src/screens/SignTransaction/SignTransactionScreen.tsx

```diff
@@ -274,6 +274,48 @@ function SignTransactionScreen() {
     }
   }, [sendFailedMessage, broadcasting]);
 
+  useEffect(() => {
+    let serverKey: VaultSigner;
+    let serverKeySigner: Signer;
+    for (const key of vaultKeys) {
+      const signer: Signer = signerMap[getKeyUID(key)];
+      if (signer.type === SignerType.POLICY_SERVER) {
+        serverKey = key;
+        serverKeySigner = signer;
+      }
+    }
+
+    if (serverKey && serverKeySigner) {
+      // Server Key is one of the signers
+      const serializedPSBTEnvelop = serializedPSBTEnvelops.filter(
```

Comment:
In Miniscript cases there may not be a PSBT for the server key, so should not just get [0] but check it exists first